src/wp-admin/includes/image-edit.php

```diff
@@ -209,61 +209,61 @@ function wp_image_editor( $post_id, $msg = false ) {
 				<fieldset class="imgedit-crop-ratio">
 					<legend><?php _e( 'Aspect ratio:' ); ?></legend>
 					<div class="nowrap">
-					<label for="imgedit-crop-width-<?php echo $post_id; ?>" class="screen-reader-text">
+					<label for="imgedit-crop-width-<?php echo esc_attr($post_id); ?>" class="screen-reader-text">
```

Comment:
```suggestion
					<label for="imgedit-crop-width-<?php echo esc_attr( $post_id ); ?>" class="screen-reader-text">
```
Change this everywhere to pass the  PHP coding standards Checks

Reply:
```suggestion
					<label for="imgedit-crop-width-<?php echo $post_id; ?>" class="screen-reader-text">
```

The `$post_id` variable must be an integer, which should not require any escaping. Both `wp_ajax_image_editor()` and `edit_form_image_editor()` cast their attachment IDs to `(int)` type.